src/style.css

```diff
@@ -1,3 +1,94 @@
-body {
+html {
+  --blue-color: #00acdc;
+}
+
+/* Скидання стандартних відступів браузера */
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Встановлюємо шрифт Roboto */
+body {
+  font-family: Roboto, sans-serif;
+}
+
+/* Стилі для заголовка */
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+}
+
+/* Логотип як посилання */
+.logo-link img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+/* Основні стилі для навігації */
+nav {
+  display: flex;
+  align-items: center;
+}
+
+/* Список навігаційних посилань */
+.style-link {
+  list-style: none;
+  display: flex;
+}
+
+.style-link li {
+  margin-right: 20px;
+}
+
+.style-link li:first-child {
+  margin-left: 0;
+}
+
+.style-link li:last-child {
+  margin-right: 0;
+}
+
+/* Стилі для навігаційних посилань */
+.nav-link {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: #000;
+  line-height: 60px;
+  display: block;
+  text-align: center;
+  position: relative;
+}
+
+/* Стилі для активного посилання */
+.nav-link.is-active {
+  color: var(--blue-color);
+}
+
+.nav-link.is-active::after {
+  content: '';
+  width: 100%;
+  position: absolute;
+  background-color: var(--blue-color);
+  height: 4px;
+  border-radius: 8px;
+  left: 0;
+  bottom: 0;
+}
+
+/* Стилі для навігаційних посилань при наведенні */
```

Comment:
and pls remove all comments
Try to write obvious code that explains all we need without comments)

Reply:
ok